sky/global_user_state.py

```diff
@@ -359,7 +540,38 @@ def get_clusters() -> List[Dict[str, Any]]:
             'to_down': bool(to_down),
             'owner': owner,
             'metadata': json.loads(metadata),
+            'cluster_hash': cluster_hash,
         }
+
+        records.append(record)
+    return records
+
+
+def get_clusters_from_history() -> List[Dict[str, Any]]:
+    rows = _DB.cursor.execute('SELECT * from cluster_history').fetchall()
```

Comment:
```suggestion
    rows = _DB.cursor.execute('SELECT * from cluster_history order by launched_at desc').fetchall()
```

Reply:
the launched at does not exist as a column in the sqlite table, it is calculate on the fly, so I can instead sort the rows in python using the launched_at after it is calculated